views/shared/modals/quest-rewards.jade

```diff
@@ -1,6 +1,6 @@
 script(id='partials/options.social.party.quest-rewards.html', type='text/ng-template')
   hr
-  h5 {{header}}
+  h5= env.t('rewards')
```

Comment:
This is not correct. It should remain `{{header}}`, the problem is actually in this file: https://github.com/HabitRPG/habitrpg/blob/154b8a00e58d1ad30f57ac1280ead9fdaeb87c88/views/shared/modals/quests.jade

As you can see in line 10, header gets declared as `env.t('youReceived')`

But the header declaration is missing for lines 34, 62 and 87. Those should be changed to have a header declaration as well.